main.tf

```diff
@@ -238,6 +238,16 @@ resource "aws_lb_target_group" "service" {
     }
   }
 
+  dynamic "stickiness" {
+    for_each = var.lb_stickiness[*]
+        content {
+      type            = var.lb_stickiness.type
+      enabled         = var.lb_stickiness.enabled
+      cookie_duration = var.lb_stickiness.type == "lb_cookie" ? var.lb_stickiness.cookie_duration : null
```

Comment:
Her sier dokumentasjonen til Load Balanceren at alb'n sjekker duration på cookien https://docs.aws.amazon.com/elasticloadbalancing/latest/application/sticky-sessions.html
Mens dokumentasjonen til aws provideren, og aws_lb_target_group sier at den er optional. Om det ikke utgjør noen forskjell for terraform, så syns jeg vi kan sette cookie_duration uansett

Reply:
`var.lb_stickiness.cookie_duration`